src/main/kotlin/com/rooplor/classcraftbackend/controllers/ClassController.kt

```diff
@@ -26,6 +27,7 @@ class ClassController
     constructor(
         val service: ClassService,
         val modelMapper: ModelMapper,
+        private val classService: ClassService,
```

Comment:
What is the reason for adding this? There's `ClassService` on top already

Reply:
fixed